.github/styles/Vocab/Docs/accept.txt

```diff
@@ -1,24 +1,64 @@
-D2iQ
-COPS
-Mesos
+addon
+Alertmanager
+Ansible
+autoscal(s|ing)
+Cassandra
+checkpoint(s|ed|ing)
```

Comment:
```suggestion
checkpoint(|s|ed|ing)
```
We want to allow `checkpoint` as well, right? Or is that accepted already?

Reply:
Good point!

Reply:
I think that should already be a word on and of itself but the verb conjugations may not be. Still, I like making it clear that all versions are accepted.